src/core/handle.ml

```diff
@@ -148,27 +160,28 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
       let a =
         match ao with
         | Some(a) ->
-            Typing.sort_type ss.builtins [] a;
-            if Typing.check ss.builtins [] t a then a else
-            fatal cmd.pos "The term [%a] does not have type [%a]." pp t pp a
+            Typing.sort_type [] a;
+            if Typing.check [] t a then a else
+              fatal cmd.pos "The term [%a] does not have type [%a]."
+                pp_term t pp_term a
```

Comment:
Using `pp` saved lines here as well.

Reply:
I prefer not to use pp. It is not explicit enough.